### ISSUE TITLE
[Accessibility] Pagination disabled links require role and state properties

### ISSUE THREAD

**Issue by aklxx100** [reactstrap/reactstrap#1548]
### Issue description

- components: `PaginationLink`
- reactstrap version `8.0.0`
- import method `es`
- react version `16.8.1`
- bootstrap version `4.0.1`

### What is happening?

Pagination is failing accessibility test `Elements must have sufficient color contrast` because disabled links are being grayed out.

### What should be happening?
`<a>` tag requires `role="button" aria-disabled="true"` properties when disabled.

```
<li class="page-item disabled">
  <a href="#" class="page-link" aria-label="Previous" role="button" aria-disabled="true">
    <span aria-hidden="true">‹</span>
    <span class="sr-only">Previous</span>
  </a>
</li>
```

### Steps to reproduce issue

Just inspect the element on the browser.

**Comment by TheSharpieOne** [comment]
We also shouldn't be generating `<a>` with `href="#"`
